src/Stubby.ts

```diff
@@ -78,7 +78,7 @@ export class Stubby {
       for(const key of listRet.keys)
         keys.push(this.kv.delete(key.name));
       if(listRet.list_complete) break;
-      cursor = listRet.cursor;
+      if(typeof(listRet.cursor) === 'string') cursor = listRet.cursor;
```

Comment:
I have opened an issue in the workers-types repo (https://github.com/cloudflare/workers-types/issues/264), until then this is a good workaround.